### PR TITLE
MINOR: Verify startup of zookeeper service in system tests

### PR DESCRIPTION
**Pull request by kkonstantine**

### CODE REVIEW

**Comment by kkonstantine** [comment]
@ewencp @hachikuji @ijuma 
This should make sure ZK service is up before Kafka service attempts to startup.

**Comment by kkonstantine** [comment]
Tested normal path (ZK starting) and abnormal path (ZK being shut down before `wait_until` check runs).

**Comment by kkonstantine** [comment]
Also, AK system tests have succeeded on top of this PR:

Report: 
http://s3-us-west-2.amazonaws.com/confluent-kafka-branch-builder-system-test-results/2017-08-21--001.1503395329--kkonstantine--MINOR-Verify-startup-of-zookeeper-service-in-system-tests-by-connecting-to-port--0e7fae5/report.html

File: tests/kafkatest/services/zookeeper.py [hunk 1]
```diff
@@ -82,7 +82,16 @@ def start_node(self, node):
...
 
-        time.sleep(5)  # give it some time to start
+        wait_until(lambda: self.listening(node) == "success", timeout_sec=10, err_msg="Zookeeper node failed to start")
+
+    def listening(self, node):
+        try:
+            cmd = "nc -z %s %s" % (node.account.hostname, 2181)
+            node.account.ssh_output(cmd, allow_fail=False)
+            self.logger.debug("Zookeeper started accepting connections at: '%s:%s')", node.account.hostname, 2181)
+            return "success"
```
Comment (ewencp) [inline tests/kafkatest/services/zookeeper.py]: This should probably just return a boolean

Comment (kkonstantine) [reply tests/kafkatest/services/zookeeper.py]: haha, oh python! my bad. Thanks!

**Comment by ewencp** [comment]
LGTM

**Pull request closed without merging**